services/teamcity/teamcity-build.service.js

```diff
@@ -24,7 +24,7 @@ module.exports = class TeamCityBuild extends TeamCityBase {
       base: 'teamcity',
       // Do not base new services on this route pattern.
       // See https://github.com/badges/shields/issues/3714
-      format: '(?:codebetter|(http|https)/(.+)/(s|e))/([^/]+)',
+      format: '(?:codebetter|(http|https)/(.+)/(s|e))/([^/]+?)',
```

Comment:
As an FYI for anyone looking at this in the review app, the badge example listings incorrectly include an `https` prefix in the `hostAndPath` param (pretty sure I was the one that caused that 😉). If you copy/paste the example values make sure to trim that leading `https`. 

I will open a PR to fix the example listings